src/super_gradients/training/models/conversion.py

```diff
@@ -129,3 +136,16 @@ def convert_from_config(cfg: DictConfig) -> str:
     out_path = models.convert_to_onnx(model=model, **cfg)
     logger.info(f"Successfully exported model at {out_path}")
     return out_path
+
+
+def onnx_simplify(onnx_path: str, onnx_sim_path: str):
+    """
+    onnx simplifier method, same as `python -m onnxsim onnx_path onnx_sim_path
+    :param onnx_path: path to onnx model
+    :param onnx_sim_path: path for output onnx simplified model
+    """
```

Comment:
Minor comment: If we return onnx_sim_path we should add typing and :return:
But I'm not sure we need/want to return it because it's just the input params. So maybe we can just remove the return line to make it more clear ?

Reply:
sure, removed in https://github.com/Deci-AI/super-gradients/pull/755/commits/94af6edeee6ca1705778884db817e11cab32ee40